dbms/src/Functions/intDiv.cpp

```diff
@@ -21,7 +21,7 @@ struct DivideIntegralByConstantImpl
     using ResultType = typename DivideIntegralImpl<A, B>::ResultType;
     static const constexpr bool allow_fixed_string = false;
 
-    static void vector_constant(const PaddedPODArray<A> & a, B b, PaddedPODArray<ResultType> & c)
+    static void vector_constant(const A * __restrict a_pos, B b, ResultType * __restrict c_pos, size_t size)
```

Comment:
NO_INLINE, I'm not sure if GCC handles inline and __restrict correctly but vectorized function without inlining sounds reasonable.